frappe/model/__init__.py

```diff
@@ -96,6 +96,7 @@
 	'idx',
 	'docstatus'
 )
+default_fields_set = set(default_fields)
```

Comment:
Don't think this belongs here. Adding more `{field_types}_{type}` in this file doesn't make a lot fo sense to me 😅